src/Entity/TaxonomyTestTrait.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
How about naming this TaxonomyCreationTrait so its in line with the others.

Reply:
I kept it as a match for core's name, but will change it to the more sensible name :)